docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,155 @@
+# Cross Chain Transfers
+
+## Introduction
+
+AVAX tokens exist on the X-Chain, where they can be traded, on the P-Chain, where they can be provided as a stake when validating the Primary Network, and on the C-Chain, where they can be used in smart contracts or to pay for gas. In this tutorial, we’ll send AVAX tokens between the X-Chain and C-Chain.
+
+## Requirements
+
+You've completed [Run an Avalanche Node](../nodes/build/run-avalanche-node-manually.md) and are familiar with [Avalanche's architecture](../overview/getting-started/avalanche-platform.md). You are familiar with and have cloned the [AvalancheJS](https://github.com/ava-labs/AvalancheJS) repo.
```

Comment:
Do they have to clone AvalancheJS? Can they just install and use it?

Reply:
They would have to clone it if they want to change their Avalanche network config

or 

A user could add the Avalanche pakcage to their project , edit the Avalanche network config in the export script 'node_modules/avalanche/exampes/avm/buildExport...', then run it with ts-node

Reply:
Please describe these in the article.